gym/envs/tests/test_envs.py

```diff
@@ -12,7 +12,12 @@ def should_skip_env_spec_for_tests(spec):
     # We skip tests for envs that require dependencies or are otherwise
     # troublesome to run frequently
 
-    # Skip mujoco tests for pull request CI
+    # Skipping if pull request
+    is_pr = os.environ.get('TRAVIS_PULL_REQUEST') != 'false' and os.environ.get('TRAVIS_PULL_REQUEST') is not None
```

Comment:
travis.yml has the following line, which invalidates the mujoco test.
```
mkdir -p ~/.mujoco
```
By testing if TRAVIS_PULL_REQUEST is set, we can properly bypass the Mujoco test